software-modules/internet/firefox/firefox.cfg

```diff
@@ -0,0 +1,31 @@
+// KEEP A COMMENT IN THE FIRST LINE
+// Enables import bookmarks form html
+pref("browser.places.importBookmarksHTML", true);
+
+// Location to import from
+pref("browser.bookmarks.file", "/usr/lib/firefox-esr/bookmarks.html");
+
+// Disable restore default bookmarks
+pref("browser.bookmarks.restore_default_bookmarks", false);
+
+// Allows unsigned extensions (the bookmarks one)
+pref("xpinstall.signatures.required", false);
```

Comment:
Offtopic, what would we need to sign it?

Reply:
The extension currently is manually built by compressing it, although the official (and only) way to add / update an extension is through submitting it to mozilla, I tried submitting & waiting for a review when I started to play with the extension but I never looked towards following the process.

On the security side I don't think there's much to be done too either for an attacker or a target.